data/actionsmap/yunohost.yml

```diff
@@ -448,6 +448,14 @@ domain:
                     help: Domain to delete
                     extra:
                         pattern: *pattern_domain
+                -r:
+                    full: --remove-apps
+                    help: Remove apps installed on the domain
+                    action: store_true
+                -y:
+                    full: --yes
```

Comment:
Only minor comment I have is that maybe we should name this --force to be consistent with other commands ? This seems to be our de-facto standard for the "don't prompt any warning and just do it"

Reply:
![image](https://user-images.githubusercontent.com/41827/104111095-bb170f00-52de-11eb-8f36-ba0d20a0c732.png)

Turns out we already have this option, it's just not exposed on the CLI API.

I'm going to expose it and use it then.

Reply:
While ... I'm not sure it's the best idea? Force will do the removal process even if the domain is unknown, which is weird. Apparently it's an option that you've added? I guess you'll have a better opinion then '^'

Reply:
Uuuuh yeah indeed, didn't realize that ... But I guess it's good enough, thanks for the change :+1: